### PR TITLE
update(mjml-spacer): documentation

### PR DESCRIPTION
**Pull request by OReshetnikov**
Confirmed in ticket #2021

### CODE REVIEW

File: packages/mjml-spacer/src/index.js [hunk 1]
```diff
@@ -40,7 +39,7 @@ export default class MjSpacer extends BodyComponent {
         <table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td height="${parseInt(
           height,
           10,
-        )}" style="vertical-align:top;height:${height};">
```
Comment (iRyusa) [inline packages/mjml-spacer/src/index.js:43]: I dont'think vertical align is needed here but height is still required

Comment (OReshetnikov) [reply packages/mjml-spacer/src/index.js:43]: Sorry:( Reverted

File: packages/mjml-spacer/src/index.js [hunk 2]
```diff
@@ -40,7 +39,7 @@ export default class MjSpacer extends BodyComponent {
...
+        )} " style="height:${height};">
```
Comment (iRyusa) [inline packages/mjml-spacer/src/index.js:42]: Can you remove the extra space in `)} "` -> `)}"`

Comment (OReshetnikov) [reply packages/mjml-spacer/src/index.js]: Done

**Pull request merged**